docs/source/conf.py

```diff
@@ -46,6 +46,7 @@
     "matplotlib",
     "mplhep",
     "scipy",
+    "sklearn",
```

Comment:
@gipert this should be `scikit-learn` not `sklearn`, no?

Reply:
oh it looks like this is module name not the pip-install argument. This is indeed what makes build docs work, as long as `kernel=None` below.